index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+     <header>
+       <nav class="navbar">
+        <a class="logo">My Logo</a>
+        <div>
+         <img class="logo-img" src="./images/Union.png" alt="menubar">
+         <ul class="desktop--view">
+          <li> <a href="#portfolio" >Portfolio </a></li>
+          <li> <a href="#about">About</a></li>
+          <li> <a href="#contact">Contact</a></li>
+        </ul>
+        </div>
+       </nav>
+       <section class="hero">
+        <h1 class="header">I’m Ryisa <br> Glad to see you! </h1>
+         <p class="section">
+            I’m a software developer! I can help you build a product,feature or 
+            website Look through some of my work and experience! If you like what 
+            you see and have a project you need coded, don’t hestiate to contact me.
+         </p>
+         <p class="footer hero-connect">LET CONNECT</p>
+           <ul class="imgs">
+             <li><a href="#"><img class="imgs" src="./images/twitter.svg" alt="twitter handle"></a></li>
+             <li><a href="#"><img class="imgs" src="./images/linkin.png" alt="twitter handle"></a></li>
+             <li><a href="#"><img class="imgs" src="./images/github.jpg" alt="github handle"></a></li>
+             <li><a href="#"><img class="imgs" src="./images/medium.jpg" alt="medium handle"></a></li>
+             <li><a href="#"><img class="imgs" src="./images/angellist.jpg" alt="angellisthandle"></a></li>
+            </ul>
+      </section>
+    </header>
+
+<!--WORK SECTION-->
+
+<!--My section-->
+<a name="portfolio">
+<section class='card-section'>
+ 
+    <div class="card"> 
+      <div class='card-img'> 
+        <img src="./images/first_img.png" alt='project image'>
+      </div>
+      <div class="card-decri">
+        <h3>Multi-post <br /> Stories</h3>
+        <div class="tags-canopy">
+          <h5>CANOPY</h5>
+          <ul class="list-tags">
+            <li> Back End Dev </li>
+            <li>2015 </li>
+          </ul>
+        </div>
+        <p> A daily selection of privately
+            personalized reads; no account or sign-up required
+        </p>
+  
+        <ul class="tag-ul">
+          <li> html </li>
+          <li> css</li>
+          <li> javascript</li>
+        </ul>
+          <div>
+          <button class="btn-card">
+            See project 
+          </button>
```

Comment:
- [ ] [OPTIONAL] I think it could be a good idea to add the effects/states for the buttons (e.g., hover, focus, active) :smiley: in order to make your portfolio look even better :smiley: